db/seeds.rb

```diff
@@ -7,7 +7,7 @@
     password: 'asdfasdf'
   )
 
-  user.confirm!
+  user.confirm
```

Comment:
Will this work the same as the version without `!`?

Reply:
Yes, Devise 4.2.0 has the confirm! method depricated, now you have to use confirm instead.